sources/packages/web/src/views/aest/institution/ViewOffering.vue

```diff
@@ -0,0 +1,151 @@
+<template>
+  <full-page-container>
+    <template #header>
+      <header-navigator
+        title="Study period offerings"
+        :routeLocation="programRoute"
+        subTitle="View Request"
+      >
+        <template #buttons v-if="showActionButtons">
+          <v-btn
+            color="primary"
+            variant="outlined"
+            @click="assessOffering(OfferingStatus.Declined)"
+            >Decline</v-btn
+          >
+          <v-btn
+            class="ml-2"
+            color="primary"
+            @click="assessOffering(OfferingStatus.Approved)"
+            >Approve offering</v-btn
+          >
+        </template>
+      </header-navigator>
+      <program-offering-detail-header
+        class="m-4"
+        :headerDetails="{
+          ...initialData,
+          status: initialData.offeringStatus,
+          institutionId: institutionId,
+        }"
+      />
+    </template>
+    <offering-form :data="initialData"></offering-form>
+    <assess-offering-modal
```

Comment:
I assume that this PR was already in progress when we took the decision to stop creating form.io modals, right?

Reply:
This modal was created for Approving/Declining pending offerings which was one or two months back. Because of the refactor it appeared it changes.